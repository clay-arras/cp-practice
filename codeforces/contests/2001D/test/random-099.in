1
17
10 7 20 2 21 21 21 14 13 10 10 18 4 19 17 2 4
