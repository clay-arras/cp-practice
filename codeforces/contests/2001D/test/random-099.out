11
10 7 20 2 21 14 13 18 19 17 4
